addons/l10n_fr_pos_cert/models/account_bank_statement.py

```diff
@@ -1,31 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import models, api
-from odoo.tools.translate import _
-from odoo.exceptions import UserError
-
-
-class AccountBankStatement(models.Model):
-    _inherit = 'account.bank.statement'
-
-    @api.ondelete(at_uninstall=True)
-    def _unlink_except_created_by_pos(self):
-        for statement in self:
-            if not statement.company_id._is_accounting_unalterable() or not statement.journal_id.pos_payment_method_ids:
-                continue
-            if statement.state != 'open':
-                raise UserError(_('You cannot modify anything on a bank statement (name: %s) that was created by point of sale operations.') % statement.name)
-
-
-
-class AccountBankStatementLine(models.Model):
```

Comment:
We need to check statement lines and prevent them from changing, we just don't need the statements themselves.
So this override should stay and only the above one should be removed

Reply:
Statement line check the statement state too
A PR already exist for this issue, I have closed this one as it's just a duplicate

Reply:
Now, statement line is independent of the statement, so you can access journal_id and company_id from the line itself. There is no state field anymore, so the line should never be modified or unlinked in this case.